### DISCUSSION TITLE
What do the range fields mean in the forecast values?

### DISCUSSION THREAD

**Discussion by micarex** [lewisbenge/hass-amber-electric#30] [Q&A] [answered]
@madpilot (or anyone else) wondering if you can provide some info on what range values mean in the forecast values.

I was going to self serve the answer but I can't find a link to the API docs on the Amber site any longer for some reason. 

So when the API returns this: 

- duration: 30
  nem_date: '2021-08-14T14:00:00+10:00'
  per_kwh: 7
  spot_per_kwh: -2
  start_time: '2021-08-14T03:30:01+00:00'
  end_time: '2021-08-14T04:00:00+00:00'
  renewables: 51
  spike_status: none
  range_min: -9.948
  range_max: 1.54336

Does this mean: the expected price within start_time and end_time is per_kwh but could vary between range_min and range_max of the per_kwh value?

**Comment by madpilot** [comment, marked as answer]
Weird. I'll check your feature flags. You can go straight to https://app.amber.com.au/developers

AEMO (the market operator) runs about 100 different models when calculating the forecast price. The forecast range is the minimum and maximum values of all the results from those models.

If the range is large, you can assume volatility. If it's small, it's a good change the final price will be close to the predicted price.

The range in the current price is different - we synthesise that based on the previous and next three forecast values.

**Comment by micarex** [comment]
Thanks for that.  I don't think there is anything wrong with my account.  I just couldn't remember the URL for the docs.  Resolved now.
